apps/web/src/pages/templates/components/LackIntegrationAlert.tsx

```diff
@@ -76,6 +83,26 @@ const WarningMessage = styled.div`
   margin-bottom: 40px;
   color: #e54545;
 
-  background: rgba(230, 69, 69, 0.15);
+  background: ${({ backgroundColor }) => backgroundColor};
   border-radius: 7px;
 `;
+
+function alertTypeToDoubleArrowColor(type: alertType) {
+  switch (type) {
+    case 'warning':
+      return 'rgba(234, 169, 0, 1)';
+    default:
+      return 'undefined';
```

Comment:
at the moment the default is the error color so no need in mapping the default will handle it.